CanvasPlusPlayground/Features/Calendar/GlobalCalendarView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  GlobalCalendarView.swift
+//  CanvasPlusPlayground
+//
+//  Created by Ethan Fox on 10/26/25.
+//
+
+import SwiftUI
+
+struct GlobalCalendarView: View {
+
+    @Environment(\.horizontalSizeClass) var horizontalSizeClass
+
+
+    @Environment(GlobalCalendarManager.self) var calendarManager
+    @Environment(CourseManager.self) var courseManager
+
+    var body: some View {
+        HStack(spacing: 0) {
+            ForEach(Array(calendarManager.currentWeekDates.enumerated()), id: \.element) { index, date in
+                if index > 0 {
+                    Divider()
+                }
+                DayColumn(date: date, events: calendarManager.calendarEvents.filter { event in
+                    Calendar.current.isDate(event.startDate, inSameDayAs: date)
+                }, backgroundTint: Calendar.current.isDateInToday(date) ? .blue.opacity(0.15) : nil)
```

Comment:
This could also be an extension convience:

```suggestion
                }, backgroundTint: date.isInToday ? .blue.opacity(0.15) : nil)
```